libs/api/api_internal.h

```diff
@@ -23,9 +23,9 @@ typedef struct _ebpf_program
     fd_t fd;
 } ebpf_program_t;
```

Comment:
Is anyone still using the ebpf_program_t typedef?

Reply:
Yes there are still a bunch of uses in code in this repo